src/Symfony/Component/Form/Tests/Extension/Validator/DataTransformer/ViewObjectTransformer.php

```diff
@@ -0,0 +1,73 @@
+<?php
+namespace Symfony\Component\Form\Tests\Extension\Validator\DataTransformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+
+/**
+* Transforms Object to array for view
+*
+* @author Charles J. C. Elling <charles@denumeris.com>
+*/
+class ViewObjectTransformer implements DataTransformerInterface {
```

Comment:
Ok, i will try to implement a less weird solution replacing the `(string) $child->getPropertyPath() line 191` for a local method call.